sdcm/cluster.py

```diff
@@ -2001,8 +2017,8 @@ def kill_stress_thread(self):
             loader.remoter.run(cmd='rm -rf %s' % kill_script_dir)
         kill_script.remove()
 
-    @staticmethod
-    def _parse_cs_summary(lines):
+    # @staticmethod
+    def _parse_cs_summary(self, lines):
```

Comment:
Why the change?
Are you sure it doesn't affect anything?

Reply:
I did it to be able to print debug info and forgot to revert before commit. Thanks